apps/vaporgui/AnnotationEventRouter.cpp

```diff
@@ -81,6 +82,12 @@ AnnotationEventRouter::AnnotationEventRouter(QWidget *parent, ControlExec *ce) :
 
     _animConnected = false;
     _ap = NULL;
+
+    _aa = new PGroup({new PSection("Orientation Arrows", {(new PCheckbox(AnnotationParams::AxisArrowEnabledTag, "Show arrows (XYZ->RGB)")),
```

Comment:
Clang-Format struggles with GUI code. You can manually format it as follows
```
// clang-format off
GUI code here
// clang-format on
```
Look at the event routers (you need to merge with master first) for an example. I don't think we've decided we're all expected to do this so its up to you or we can discuss in our meeting, but I think it makes it more readable.

Reply:
Sounds good. Will do and note when it's done.